File: SiyasMatrimony.API/Models/Country.cs
using System.Collections.Generic;

namespace SiyasMatrimony.API.Models
{
    public class Country
    {
        public int id { get; set; }
        public string name { get; set; }
        public string iso3 { get; set; }
        public string iso2 { get; set; }
        public string phone_code { get; set; }
        public string capital { get; set; }
        public string currency { get; set; }
        public ICollection<State> States { get; set; }
        public ICollection<City> Cities { get; set; }
    }
}

File: SiyasMatrimony-SPA/src/app/_models/Country.ts
import { State } from './State';
import { City } from './City';

export interface Country {

     id: number;
     name: string;
     iso3: string;
     iso2: string;
     phone_code: string;
     capital: string;
     currency: string;
     States: State[];
     Cities: City[];
}
